.github/workflows/ci.yml

```diff
@@ -1,25 +1,22 @@
 name: CI
 
 on:
-  pull_request:
-    branches: ['*']
-  merge_group:
-    types: [checks_requested]
```

Comment:
akurat wy to chyba chcecie?

Reply:
raczej tak (chyba że eksperyment z merge queue uznajemy za nieudany)

Reply:
wyłączyłem ostatnio merge queue, bo psuje linear history